test/test_individual.py

```diff
@@ -3,21 +3,46 @@
 import pytest
 
 import cgp
-from cgp.individual import IndividualSingleGenome
+from cgp import Genome, IndividualSingleGenome, IndividualMultiGenome
 from cgp.genome import ID_INPUT_NODE, ID_OUTPUT_NODE, ID_NON_CODING_GENE
 
 
-def test_pickle_individual():
+def _create_individual(fitness, genome, ind_type="single"):
+    if ind_type == "single":
+        return IndividualSingleGenome(fitness, genome)
+    elif ind_type == "multi":
+        if isinstance(genome, Genome):
```

Comment:
why do we need this `if`? shouldn't there be only one way to use this function to create `IndividualMultiGenome` either with genome being a `Genome` or `[Genome]`?

Reply:
I guess, I was being overly cautious or wanted to prepare possible future cases where we could also pass a list of Genome. Removed.

Reply:
44f5d161a2fcdf1c96c6afc278c37b683f7f3675